optimum/habana/transformers/training_args.py

```diff
@@ -123,6 +125,11 @@ class GaudiTrainingArguments(TrainingArguments):
         },
     )
 
+    throughput_rm_save_ckpt_time:  bool = field(
+        default=False,
+        metadata={"help": "Whether to remove save checkpoint time from throughput calculation."},
+    )
```

Comment:
```suggestion
    adjust_throughput:  bool = field(
        default=False,
        metadata={"help": "Whether to remove the time taken for logging, evaluating and saving from throughput calculation."},
    )
```
A proposal to take into account everything.